include/revng/ADT/Concepts.h

```diff
@@ -125,3 +125,11 @@ template<typename T, typename... Types>
 inline constexpr bool anyOf() {
   return (std::is_same_v<T, Types> || ...);
 }
+
+template<typename T>
+concept isUint = anyOf<T, uint8_t, uint16_t, uint32_t, uint64_t>();
```

Comment:
1. Capitalize all `concept`s.
2. I'd go with `std::is_integral_v`. Why not handle signed types as well?

Reply:
1. Ok
2. `std::is_integral` also includes other types which are not desirable, such as `bool`, `char*_t`, `char` and `wchar_t`. Handling signed types can be added, but I wanted to be strict on the types supported. If need be a `IsInt` and `IsIntOrEnum` can be used